ztp/gitops-subscriptions/argocd/example/siteconfig/hp.profile

```diff
@@ -0,0 +1,6 @@
+BootMode: Uefi
```

Comment:
Wouldn't we drop this in as extra manifests

Reply:
Yes, that's another option.

Reply:
I can move it to extra-manifest.

Reply:
The extra-manifests are used to generate the ConfigMap that is consumed by assisted-service to populate the MachineConfig for the spoke clusters. The BIOS configuration file is used to generate the ConfigMap that is consumed by baremetal-operator to populate the Spec fields.
It is prudent to take a closer look at this option.